templates/src/serialize.c.erb

```diff
@@ -47,6 +47,21 @@ pm_serialize_string(pm_parser_t *parser, pm_string_t *string, pm_buffer_t *buffe
     }
 }
 
+/**
+ * Serialize a 32-bit integer to the given address always in little-endian.
+ */
+static void
+pm_serialize_32(char *address, uint32_t value) {
+    if (PRISM_WORDS_BIGENDIAN) {
```

Comment:
```suggestion
#ifdef PRISM_WORDS_BIGENDIAN
```
So the C compiler doesn't even see the code for the wrong endian.

Reply:
👍